.circleci/config.yml

```diff
@@ -20,24 +20,70 @@ aliases:
        echo "*** Octave test result: "${RESULT}
        exit $RESULT
 
+  - &run_helics_tests
+    name: run_helics_tests
+    command: |
+       SUMRESULT=0
```

Comment:
would it be possible to put this in a script? it is likely in the future there will be a few additional tests that need to be added,  It would simplify future maintenance if it were in a script vs purely in the .yml.

Also might be useful to have this be in a for each loop and then a list of the tests?

Reply:
allright, but I will delete the intermediate messages such as
```
       echo "*** Helics core-test result: "${RESULT}
```
and maybe just 
```
       echo "*** last results: ${RESULT}"   
```

I'll try to echo the command line so that you know which one past/failed.